inc/theme/namespace.php

```diff
@@ -167,6 +167,33 @@ function migrate_book_themes() {
 		$pressbooks_theme_migration = 3;
 		update_option( 'pressbooks_theme_migration', $pressbooks_theme_migration );
 	}
+
+	// Transition from Dillard + Dillard Plain 1.x to Dillard 2.0
+	if ( $pressbooks_theme_migration === 3 ) {
+		$theme = wp_get_theme()->get_stylesheet();
+		if ( $theme === 'pressbooks-dillard' ) {
+			// Turn on title decoration
+			$options = get_option( 'pressbooks_theme_options_global' );
+			$options['enable_title_decoration'] = 1;
+			update_option( 'pressbooks_theme_options_global', $options );
+		}
+
+		if ( $theme === 'pressbooks-dillardplain' ) {
+			// Migrate to Dillard 2.0
+			switch_theme( 'pressbooks-dillard' );
+			$lock = Lock::init();
+			if ( $lock->isLocked() ) {
```

Comment:
Can I get a code comment here? I'm not sure what `Migrate to Dillard 2.0` followed by some lock picking is doing.